apps/src/lib/node/ledger/shims/abcipp_shim.rs

```diff
@@ -49,7 +49,6 @@ impl Service<Req> for AbcippShim {
     }
 
     fn call(&mut self, req: Req) -> Self::Future {
-        tracing::debug!(?req);
```

Comment:
I removed this for now as some of the requests are big (including tx bytes). If it's still useful, we can maybe just log the request type and other stuff in the cases